<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadowPlotterGUI</class>
 <widget class="QMainWindow" name="ShadowPlotterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Shadow Plotter</string>
  </property>
  <widget class="QWidget" name="central_area">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="main_grid">
      <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QWidget" name="dia_plot" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QWidget" name="image_plot" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="up_grid">
        <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
         <widget class="QWidget" name="up_plot" native="true">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="right_grid">
        <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QWidget" name="right_plot" native="true">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the main application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
